yahoo-japan/land-and-water.json

```diff
@@ -0,0 +1,103 @@
+{
+  "type": "FeatureCollection",
+  "name": "land-and-water",
```

Comment:
Locations look good! I think this is missing any water points though?

Reply:
Ah let me include some rivers. Only had lake Tazawa.